tutorials/scripting/cross_language_scripting.rst

```diff
@@ -43,6 +47,7 @@ The following two scripts will be used as references throughout this page.
         public string myField = "bar";
 
         [Signal] public delegate void MySignalEventHandler();
+        [Signal] public delegate void MySignalWithParamsEventHandler();
```

Comment:
You forgot to add the parameters to the signal.

```suggestion
        [Signal] public delegate void MySignalWithParamsEventHandler(string msg, int n);
```

Reply:
Fixed